orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/support/CustomTriggerDeserializerSupplier.kt

```diff
@@ -29,6 +29,9 @@ import com.netflix.spinnaker.orca.api.pipeline.models.Trigger
  * TODO(rz): Refactor so that JsonNode is not needed (for orca-api compat)
```

Comment:
lol at this comment

but yea, hence why I wanted to use something that was higher level than `JsonNode`.